pkg/controller/migmigration/description.go

```diff
@@ -0,0 +1,53 @@
+package migmigration
+
+// Phase Descriptions
+var PhaseDescriptions = map[string]string{
+	"Started":                         "Migration started.",
+	"Prepare":                         "Preparing for migration.",
+	"EnsureCloudSecretPropagated":     "Ensuring Velero has latest Replication Repository storage credentials.",
+	"PreBackupHooks":                  "Waiting for user-defined pre-backup hooks to complete.",
+	"PostBackupHooks":                 "Waiting for user-defined post-backup hooks to complete.",
+	"PreRestoreHooks":                 "Waiting for user-defined pre-restore hooks to complete.",
+	"PostRestoreHooks":                "Waiting for user-defined post-restore hooks to complete.",
+	"PreBackupHooksFailed":            "Migration failed while running user-defined pre-backup hooks.",
+	"PostBackupHooksFailed":           "Migration failed while running user-defined post-backup hooks.",
+	"PreRestoreHooksFailed":           "Migration failed while running user-defined pre-restore hooks.",
+	"PostRestoreHooksFailed":          "Migration failed while running user-defined post-restore hooks.",
+	"EnsureInitialBackup":             "Creating initial Velero backup.",
+	"InitialBackupCreated":            "Waiting for initial Velero backup to complete.",
+	"InitialBackupFailed":             "Migration failed during initial Velero backup.",
+	"AnnotateResources":               "Adding migration annotations and labels to PVs, PVCs, Pods, ImageStreams, and Namespaces. Annotations and labels provide migration instructions to Velero, Velero Plugins and Restic.",
+	"EnsureStagePodsFromRunning":      "Creating Stage Pods and mounting PVC data from running Pods.",
+	"EnsureStagePodsFromTemplates":    "Creating Stage Pods and mounting PVC data from DeploymentTemplates, DeploymentConfigs, ReplicationControllers, DaemonSets, StatefulSets, ReplicaSets, CronJobs and Jobs.",
```

Comment:
Suggest rewording this to: "Creating Stage Pods and mounting PVC using pod templates from Deployments, DeploymentConfigs, ReplicationControllers, DaemonSets, StatefulSets, ReplicaSets, CronJobs and Jobs."

Reply:
👍  on the change, just need to capitalize `Pod` per UX/docs team guidance on these descriptions.